### PR TITLE
feat: 0.8.2b0

### PR DESCRIPTION
**Pull request by zhouaihui**
### Added
- TEE python Unit(TEEU) is introduced as the TEE cryptographic device and enables authorized computation with authorized data in TEE. TEEU brings more possibilities for hybrid computation.
- An experimental SecretFlow component design.
- SGB feature: add support for pre-pruning and model save & load.

### Changed
- Use pytest instead of unittest.
- Bump spu to 0.3.3b0
- Bump heu to 0.4.3b3

### Fix
- Fix hess lr auc err with large learning_rate.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [0.8.2b0] - 2023-4-19
+### Added
+- TEE python Unit(TEEU) is introduced as the TEE cryptographic device and enables authorized computation with authorized data in TEE. TEEU brings more possibilities for hybrid computation.
+- An experimental SecretFlow component design.
+- SGB feature: add support for pre-pruning and model save & load.
+
+### Changed
+- Use pytest instead of unittest.
+- Bump spu to 0.3.3b0
+- Bump heu to 0.4.3b2
```
Comment (anakinxc) [inline CHANGELOG.md:31]: spu 0.3.3b0
heu 0.4.3b3

**Review by anakinxc — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,36 @@
 # Contribution guide
 
 ## Build
-Please check [INSTALLATION.md](./INSTALLATION.md) to build SecretFlow from source.
+Please check [INSTALLATION.md](./docs/getting_started/installation.md) to build SecretFlow from source.
```
Comment (ian-huu) [inline CONTRIBUTING.md:4]: INSTALLATION.md

Comment (zhouaihui) [reply CONTRIBUTING.md:4]: It's lowercase now.

File: CHANGELOG.md (hunk 1 above)
Comment (zhouaihui) [reply CHANGELOG.md:31]: Updated.

File: docs/tutorial/Split_Learning_for_bank_marketing.ipynb [hunk 3]
```diff
@@ -95,18 +95,13 @@
   {
    "cell_type": "code",
    "execution_count": 1,
```
Comment (ian-huu) [inline docs/tutorial/Split_Learning_for_bank_marketing.ipynb:97]: this tutorial seems to be overwritten

Comment (zhouaihui) [reply docs/tutorial/Split_Learning_for_bank_marketing.ipynb:97]: @zhangxingmeng refactored this tutorial.

**Review by anakinxc — DISMISSED** [review]
LGTM

**Review by ian-huu — DISMISSED** [review]
LGTM

File: CONTRIBUTING.md (hunk 2 above)
Comment (zhouaihui) [reply CONTRIBUTING.md:4]: Updated.

**Review by ian-huu — APPROVED** [review]

**Pull request merged**